public_html/administrator/components/com_jed/views/review/tmpl/edit.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
This file has no use in the backend as far as I know. In front-end we use it to create menu items.